readme.markdown

```diff
@@ -221,8 +221,8 @@ instead of a name.
 ### nimble uninstall
 
 The ``uninstall`` command will remove an installed package. Attempting to remove
-a package which other packages depend on is disallowed and will result in an
-error. You must currently manually remove the reverse dependencies first.
+a package which other packages depend on will result in an error. You can use the
+``--deps`` or ``-d`` flag to remove all reverse dependencies along with the package.
```

Comment:
I was wondering about this as well. The message can be improved to say `dependent packages` instead of `reverse dependencies`. What do you prefer for a flag name? `--incldeps` perhaps? Is `-d` okay or do you prefer `-i`?

Reply:
I've updated the flag to `-i` and `--inclDeps` for now.